imports/server/schemas/DriveActivityLatest.ts

```diff
@@ -0,0 +1,23 @@
+import * as t from 'io-ts';
+import { date } from 'io-ts-types';
+import { Overrides, buildSchema } from '../../lib/schemas/typedSchemas';
+
+const DriveActivityLatestCodec = t.type({
+  _id: t.string,
```

Comment:
Please add a comment that this is expected to be a singleton collection, and that the single known `_id` should be `default`.